main_test.go

```diff
@@ -805,6 +805,38 @@ func testPkg(t *testing.T, table pkg) {
 	}
 }
 
+func copyPackage() (string, error) {
+	var err error
+	pkgCopyDir, err := ioutil.TempDir("", "gopy-")
+	if err != nil {
+		return "", err
+	}
+	target := filepath.Join(pkgCopyDir, "src", "github.com", "go-python", "gopy")
+	if err = os.MkdirAll(target, 0744); err != nil {
+		os.RemoveAll(pkgCopyDir)
+		return "", err
+	}
+	cmd := exec.Command("cp", "-r", ".", target)
+	if err := cmd.Run(); err != nil {
+		os.RemoveAll(pkgCopyDir)
+		return "", err
+	}
+	return pkgCopyDir, nil
+}
+
+func TestMain(m *testing.M) {
+	pkgCopyDir, err := copyPackage()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to install copy of gopy in a temp dir: %v", err)
+		os.Exit(1)
+	}
+	testEnvironment = append(testEnvironment, "GOPATH="+pkgCopyDir)
```

Comment:
perhaps add a `FIXME` mentioning we should get rid of this in favor of full Go modules support?
(and/or file an issue about that?)

Reply:
Sorry, I neglected windows! But rather than firing up a vm to test things with the correct incantation of copy I went back to my original approach of writing a fake go.mod. It seems that the documentation for go modules is a little misleading and that a require statement must still accompany a replace, at least for goimports to work. (The docs mention a problem in 1.11 that was fixed in 1.12+ but I think goimports works differently). Anyway, the go.mod approach works now!